src/components/Timer/subcomponents/TimerView.tsx

```diff
@@ -116,9 +132,12 @@ export default function TimerView(props: {
               ref={minutesInputRef}
               type="number"
               className="timer-digits"
+              disabled={props.isTimerRunning}
               value={minutesInput}
               onBlur={resetMinutes}
               onChange={onMinutesChange}
+              onKeyDown={onMinutesKeyDown}
+              onClick={(e) => e.currentTarget.select()}
```

Comment:
Perhaps this should be `onFocus`? Maybe the other functions don't need it then.

Reply:
The behaviors are slightly different – using onClick doesn’t allow you to click twice to set your cursor at a specific position, it will always force a full selection and full retyping. I do get that it’s unintuitive in a way, but also this is how pretty much all phone clock/timer apps do it (on iPhones, you even have to retype the whole thing every time you make changes – on a Samsung, you always retype either hours or minutes). And that’s the closest pattern, basically. idk, i’ll experiment further.